docset/windows/appx/Get-NonRemovableAppsPolicy.md

```diff
@@ -0,0 +1,169 @@
+---
+author: romaclac
+description: Use this topic to help prevent the uninstall of specific Windows apps with Windows PowerShell.
+external help file: Microsoft.Windows.Appx.PackageManager.Commands.dll-help.xml
+keywords: powershell, cmdlet, msix, appx, block uninstall
+manager: jvintzel
+Module Name: Appx
+ms.assetid: 
+ms.author: romaclac
+ms.date: 02/05/2020
+ms.mktglfcycl: manage
+ms.prod: w10
+ms.reviewer:
+ms.sitesec: library
+ms.technology: powershell-windows
+ms.topic: reference
+online version:
+schema: 2.0.0
+title: Get-NonRemovableAppsPolicy
+---
+
+# Get-NonRemovableAppsPolicy
+
+## SYNOPSIS
+Gets a list of the app packages that are installed and configured as non-removable apps.
+
+## SYNTAX
+
+```powershell
+Get-NonRemovableAppsPolicy 
+    -Path <string> 
+    [-WindowsDirectory <string>] 
+    [-SystemDrive <string>] 
+    [-LogPath <string>]
+    [-ScratchDirectory <string>] 
+    [-LogLevel {Errors | Warnings | WarningsInfo}]  
+    [<CommonParameters>]
+```
+
+```powershell
+Get-NonRemovableAppsPolicy 
+    -Online 
+    [-WindowsDirectory <string>] 
+    [-SystemDrive <string>] 
+    [-LogPath <string>] 
+    [-ScratchDirectory <string>] 
+    [-LogLevel {Errors | Warnings | WarningsInfo}]  
+    [<CommonParameters>]
+```
+
+## DESCRIPTION
+The **Get-NonRemovableAppsPolicy** cmdlet gets a list of the app packages what are installed, and have been configured as non-removable (can not be uninstalled). An app package has a .msix or .appx file name extension.
+
+## EXAMPLES
+
+### Example 1: Get all installed non-removable app packages
+```
+PS C:\> Get-NonRemovableAppsPolicy -Online
+```
+
+This command gets information about all installed app packages which have been previously configured as non-removable.
+
+### Example 2: Get all non-removable apps from an offline windows image
+```
+PS C:\> Get-NonRemovableAppsPolicy -Path ".\wim\image.wim"
+```
+
+This command gets all apps packages that have been loaded into the offline operating system image.
+
+## PARAMETERS
+
+### -Online
+Indicates that the cmdlet operates on a running operating system on the local host.
+
+|||
+|-|-|
+| Type: | SwitchParamter |
+| Position: | Named |
+| DefaultValue: | None |
+| Accept pipeline input: | True (ByPropertyName) |
+| Accept wildcard characters: | False |
+
+### -Path
+Specifies the full path to the root directory of the offline Windows image that you will inquire into. If the directory named Windows is not a subdirectory of the root directory, WindowsDirectory must be specified.
```

Comment:
Remove the leading space.

Reply:
I'm not sure which space you are referring to?